samples/client/petstore/typescript-angular-v2/default/.swagger-codegen/VERSION

```diff
@@ -1 +1 @@
-2.3.0
\ No newline at end of file
+2.3.1-SNAPSHOT
```

Comment:
Is this because someone forgot to update the samples? Are you sure that your branch is based on current master?

Reply:
This should be for 2.3.1 now shouldn't it @TiFu .  I thought at some point I talked to @wing328  about a patch release but I can't remember exactly.

Reply:
@kenisteward @TiFu the branch is based on the master of some 5 days ago, but I will rebase and generate the code again.